### PR TITLE
[WIP] EPMLSTRCMW-274 Create test implementations for Aggregation Auth ProjectFile and ProjectVersioning services

### PR DESCRIPTION
**Pull request by Nikolai-Isaev**
test: Create Service test implementations

Created: AggregationServiceTestImpl, AuthServiceTestImpl, ProjectFileServiceTestImpl, ProjectVersioningTestImpl

Resolves: #274

### CODE REVIEW

File: services/src/test/scala/cromwell/pipeline/service/AggregationServiceTestImpl.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package cromwell.pipeline.service
+import cromwell.pipeline.datastorage.dto._
+
+import scala.concurrent.Future
+
+class AggregationServiceTestImpl(valToReturn: DummyToReturn) extends AggregationService {
+
+  override def aggregate(run: Run): Future[CromwellInput] =
+    valToReturn match {
+      case WithException(exc)                   => Future.failed(exc)
```
Comment (DruzhininPavel) [inline services/src/test/scala/cromwell/pipeline/service/AggregationServiceTestImpl.scala:10]: Why are you writing 'exc' instead for 'exception' or 'error' ?

**Review by DruzhininPavel — COMMENTED** [review]
Nice. But I have one question in comments.

File: services/src/test/scala/cromwell/pipeline/service/AggregationServiceTestImpl.scala (hunk 1 above)
Comment (Nikolai-Isaev) [reply services/src/test/scala/cromwell/pipeline/service/AggregationServiceTestImpl.scala:10]: Takes less space ) I thought that "exc" is a well known and quite widespread abbreviation.

Comment (DruzhininPavel) [reply services/src/test/scala/cromwell/pipeline/service/AggregationServiceTestImpl.scala:10]: you can write just 'e' but however.

**Review by DruzhininPavel — APPROVED** [review]
I Have no questions.

**Pull request closed without merging**